StringCalculator/StringCalculator.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace StringCalculator;
+public class StringCalculator
+{
+    private string[] seperators = new string[] { ",", "\\n" };
+    private List<string> negativeNumbers = new List<string>();
```

Comment:
Same goes for this - the class doesn't care about the negative numbers list, only the method that deals with them should (especially since you're just going to throw an exception with those values anyway).